boards/posix/nrf52_bsim/bstests_entry.c

```diff
@@ -233,5 +233,13 @@ uint8_t bst_delete(void)
 		test_list_top = tmp;
 	}
 
+	if (bst_result == In_progress) {
+		bs_trace_raw_time(2, "TESTCASE NOT PASSED at exit (test return "
+				  "(%u) indicates it was still in progress)\n", bst_result);
+	} else if (bst_result != Passed) {
+		bs_trace_raw_time(2, "The TESTCASE FAILED (test return code %u)\n",
+				  bst_result);
+	}
```

Comment:
Would it make sense to also print the name of the test case (i.e. the test case ID)?

Reply:
It may be marginally nicer; note bs_tests only run 1 test, and you can see what script you are running in the error from the caller (or you just called it yourself). 
But it is also more work :)